vtexdocs/devportal
# Fix/contributors component

#### What is the purpose of this pull request?

Update the Contributors component to render information from file commits.

#### What problem is this solving?

This component is currently static. This PR purposes change this component and the guides page to extract authors' information using the Octokit API.

#### How should this be manually tested?

Go to this page and check if the contributors component renders the author's icons with the expected tooltip and, when clicked, takes you to the author's GitHub profile.

#### Screenshots or example usage

https://user-images.githubusercontent.com/42784961/208952086-b80b8a95-cb71-41cc-947f-54168719242c.mp4

#### Types of changes
